suites/pacific/rgw/tier-2_rgw_test-bucket-notifications.yaml

```diff
@@ -235,6 +235,17 @@ tests:
         config-file-name: test_bucket_notification_kafka_broker_multipart.yaml
         timeout: 300
 
+  - test:
+      name: put and get bucket notification using tenant user
+      desc: put and get bucket notification using tenant user
+      module: sanity_rgw.py
+      polarion-id: CEPH-83575500
+      config:
+        run-on-rgw: true
+        script-name: test_bucket_notifications.py
+        config-file-name: test_bucket_notification_with_tenant_user.yaml
+        timeout: 300
+
```

Comment:
@anrao19 , Would it be possible to have at least one suite log (single site) for the integration?

Reply:
@viduship  both logs provide is of suite log , just pointing to TC
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-0cp62/
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-gp0z9/

Reply:
@anrao19 , alright, thank you.